CMakeLists.txt

```diff
@@ -271,12 +273,13 @@ add_executable(experiment
   test/util/graph_gen.cpp
   graph.cpp
   supernode.cpp
+  work_queue.cpp
   graph_worker.cpp
   l0_sampling/sketch.cpp
   l0_sampling/update.cpp
   util.cpp)
 target_link_libraries(experiment PRIVATE xxHash::xxhash GTest::gtest)
-target_compile_definitions(experiment PRIVATE USE_FBT_F)
+# target_compile_definitions(experiment PRIVATE USE_FBT_F)
```

Comment:
so are our experiments not gonna use FBT?

Reply:
The branch where I actually have the experiments uses two executables one with FBT and one without.

Reply:
so is this executable useless?

Reply:
For this branch it's fine